### ISSUE TITLE
 No such file or directory: '0.traj'

### ISSUE THREAD

**Issue by htthYjh** [Shathra/comparing-trajectory-clustering-methods#4]
When i am running merger.py  ,some error like this: FileNotFoundError: [Errno 2] No such file or directory: '0.traj'

I have readed your other Issues ,can you show some example  of 0.traj file ,please . 

Thank you very mach

**Comment by seljukgulcan** [comment]
Hi htthYjh,

First of all, I strongly suggest looking at the [demo file](https://github.com/Shathra/comparing-trajectory-clustering-methods/blob/master/demo/demo.ipynb) first. The demo file shows the fundamental steps of trajectory clustering. Hopefully, it should be enough. Or, you can take a look at the report as well.

I was not very experienced with trajectory clustering and python in general when I was doing this project. So, the original code is quite messy, undocumented, and hard to follow. I don't remember the format of the .traj files exactly. However, from other modules, I infer that it is a dictionary of a list of tuples. I give an example:


```
{
000001: {'path':[(36, 42, 1, 5), (36.1, 42, 1, 5), (36.2, 42, 1, 5)]}
000002: {'path':[(36, 42, 1, 5), (36, 42.1, 1, 5), (36, 42.2, 1, 5)]}
}
```

each path has the following format `[(latitude_1, longitude_1, speed_1, azimuth_1), (latitude_2, longitude_2, speed_2, azimuth_2), ...]`. I don't quite remember the units of the fields.

I hope, this answer solves your issue. I'm planning to write a blog post by rewriting some parts of the demo file and giving more detail for different situations since the original code creates a considerable amount of confusion.
